test/libsolidity/semanticTests/viaYul/delete.sol

```diff
@@ -17,13 +17,24 @@ contract C {
 
 		return func() == internal_func();
 	}
-	function external_func() external pure returns (int8)
+	function delete_memory_array() public returns (bool ret)
```

Comment:
Please put this in its own test or see whether it already exists in EndToEndTests

Reply:
There is one more complex one with nested arrays. Not sure we support those already. I'll put it in its own file for now.